apps/openchallenges/app/.eslintrc.json

```diff
@@ -10,15 +10,15 @@
           "error",
           {
             "type": "attribute",
-            "prefix": "openchallenges",
```

Comment:
#Reverting to the default value used by the Angular code generator for the sake of simplification for the next time we need to regenerate the app. This means that less changes will need to be make to the `index.html` and `app.component.ts` too.